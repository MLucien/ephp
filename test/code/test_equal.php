<?php

$data = [ 1,2,3 ];
$str = "Array";

print "is equal an array to the 'Array' string? ";
print(($data == $str) ? "yes" : "no");
print "\n";

