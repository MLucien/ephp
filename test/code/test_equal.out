is equal an array to the 'Array' string? no
